### PR TITLE
Feature/merge from master

### PR DESCRIPTION
**Pull request by apapia**
Merge in upstream changes to cogito-build

### CODE REVIEW

**Comment by apapia** [comment]
I've tested this on minikube on my laptop and reviewed the merge and it looks good to me.  The GitHub action test failures looks to be because of a new security vulnerability discovered with GitHub Actions so they've disabled a feature that the test requires.  I'm going to go ahead and merge this.

**Comment by apapia** [comment]
Actually looks like I need an approver to merge.

**Review by frankjkelly — APPROVED** [review]
LGTM with one note (they use 'latest' in one docker image tag!)

File: Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ RUN cd /src && \
     echo "$(git rev-parse HEAD)" > /src/BUILD_INFO
 
 # final stage
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```
[the remark this answers is not in the document]
Comment (apapia) [reply Dockerfile:11]: yeah I actually liked this change so we pick up any CVE fixes whenever we build a new image

**Review by jgoddard-cogito — APPROVED** [review]

**Pull request merged**